contributors/devel/sig-testing/testing-strategy.md

```diff
@@ -0,0 +1,99 @@
+## Defining a Robust Testing Strategy
+
+This document outlines a testing strategy for Kubernetes features based on the **testing pyramid**, with considerations on the existing CI system's characteristics.
+
+### The Testing Pyramid
+
+Prioritize tests based on the testing pyramid, refer to the [Testing Guide](./testing.md):
+
+- **Unit Tests:**  The foundation. Fast, isolated, and cover individual components.
```

Comment:
```suggestion
- **Unit Tests:**  The foundation. Fast (a package *should* be able to run all its unit tests in seconds), isolated (doesn't depend on environment or non-local network), and cover individual components.
```

Reply:
Also doesn't depend on special permissions (root) 
(something we've repeatedly failed at and only somewhat guard now by running CI as a particular non-root user)

Reply:
Should this be 'isolated/hermetic'? (I'm finding two different references for isolated: similar to hermetic where it doesn't depend on external factors, and isolated from other tests).

Reply:
I can add both isolated and hermetic, it seems to fit into Jordans suggestion